Source/engine/render/text_render.cpp

```diff
@@ -147,6 +147,11 @@ void GetFontPath(GameFontTables size, uint16_t row, string_view ext, char *out)
 	*fmt::format_to(out, R"(fonts\{}-{:02x}.{})", FontSizes[size], row, ext) = '\0';
 }
 
+void GetFontBaseNamePath(GameFontTables size, uint16_t row, char *out)
+{
+	*fmt::format_to(out, R"(fonts\{}-{:02x})", FontSizes[size], row) = '\0';

```

Comment:
I'm guessing `GetFontPath` is no longer used with this? Let's delete it?

Reply:
`GetFontBaseNamePath()` is used for PCX and `GetFontPath()` is used for clx because the  `loaderPcx()` appends it's own extension, unlike `loadpcx()`. This is a biproduct of the unpacked mpq support.

Reply:
I see. Let's remove the `.` from the format string in `GetFontPath` and pass the extension with the dot instead (e.g. ".bin").
Then we won't need `GetFontBaseNamePath`, as we can just pass `/*ext=*/""`.